protocol/x/vest/keeper/keeper_test.go

```diff
@@ -164,6 +164,20 @@ func TestProcessVesting(t *testing.T) {
 			expectedTreasuryBalance: sdk.NewInt(3_007),
 			expectedVesterBalance:   sdk.NewInt(2_001_993),
 		},
+		"vesting in progress,  start_time < prev_block_time < block_time < end_time, vester has empty balance": {
```

Comment:
```suggestion
		"vesting in progress: expect 0 balance,  start_time < prev_block_time < block_time < end_time, vester has empty balance": {
```

Reply:
Personally, I prefer `vesting in progress,  start_time < prev_block_time < block_time < end_time, vester has empty balance` is pretty clear and more concise